docs/mismatchfinder-api-spec.yml

```diff
@@ -38,6 +38,24 @@ paths:
         '422':
           $ref: '#/components/responses/ValidationErrors'
 
+    put:
```

Comment:
shouldn't this have the {id} parameter in the URL?

```
  /mismatches/{id}:
    put:
      tags: ['store']
      summary: Update Mismatch review status
      parameters:
        - in: path
          name: id
          schema:
            type: number
          required: true

```

Reply:
:facepalm: It for sure should...